cmd/keeper/keeper.go

```diff
@@ -1160,5 +1160,7 @@ func keeper(cmd *cobra.Command, args []string) {
 	}
 	go p.Start()
 
-	<-end
+	if err := <-end; err != nil {
```

Comment:
`Receiving from a nil channel blocks forever.` Here says about nil channel, not about values received from channel. Working code.